app/src/main/java/com/elavon/converge/config/LoadBusinessIntentService.java

```diff
@@ -66,12 +68,16 @@ protected void onHandleIntent(Intent intent) {
         }
     }
 
-    private void loadBusiness(){
+    private void loadBusiness() {
```

Comment:
hmm this means you would lose the business data. Remember Business ProcessData is not complete business data. So keep it separate even though the underlying object is still business.

Reply:
code updated.